database/sqlite/sqlite_db_migration.c

```diff
@@ -148,6 +148,40 @@ static int do_migration_v5_v6(sqlite3 *database, const char *name)
     return init_database_batch(database, DB_CHECK_NONE, 0, &database_migrate_v5_v6[0]);
 }
 
+static int do_migration_v6_v7(sqlite3 *database, const char *name)
+{
+    UNUSED(name);
+    info("Running \"%s\" database migration", name);
+
+    char sql[256];
+
+    int rc;
+    sqlite3_stmt *res = NULL;
+    snprintfz(sql, 255, "SELECT name FROM sqlite_schema WHERE type ='table' AND name LIKE 'aclk_alert_%%';");
+    rc = sqlite3_prepare_v2(database, sql, -1, &res, 0);
+    if (rc != SQLITE_OK) {
+        error_report("Failed to prepare statement to alter aclk_alert tables");
+        return 1;
+    }
+
+    while (sqlite3_step_monitored(res) == SQLITE_ROW) {
+         char *table = strdupz((char *) sqlite3_column_text(res, 0));
+         if (!column_exists_in_table(table, "filtered_alert_unique_id")) {
+             snprintfz(sql, 255, "ALTER TABLE %s ADD filtered_alert_unique_id", table);
```

Comment:
Should we run a `VACUUM FULL` to rewrite the whole table? Or does sqlite rewrite by default?

Reply:
Hi Thiago! We set auto_vacuum for the whole db -> https://github.com/netdata/netdata/blob/a274b73f26221039c4d9721c5935393c1660c881/database/sqlite/sqlite_functions.c#L407-L410 I think it should be enough?

Reply:
So as mentioned in today's meeting, we don't actually vacuum the DB. But I think that's a different matter, not specific to this PR.